packages/react/src/components/MenuButton/MenuButton.tsx

```diff
@@ -0,0 +1,305 @@
+import * as React from 'react'
+import * as PropTypes from 'prop-types'
+import * as _ from 'lodash'
+import * as customPropTypes from '@stardust-ui/react-proptypes'
+
+import {
+  AutoControlledComponent,
+  RenderResultConfig,
+  applyAccessibilityKeyHandlers,
+  getOrGenerateIdFromShorthand,
+  commonPropTypes,
+  StyledComponentProps,
+} from '../../lib'
+import { ShorthandValue, ComponentEventHandler } from '../../types'
+
+import { Accessibility } from '../../lib/accessibility/types'
+import { createShorthandFactory } from '../../lib/factories'
+import Popup, { PopupProps, PopupEvents, PopupEventsArray } from '../Popup/Popup'
+import { Menu, MenuItemProps, MenuProps, Ref } from '../..'
```

Comment:
Please use direct imports

```suggestion
import { Menu, MenuItemProps, MenuProps } from '../Menu/Menu'
import { Ref } from '@stardust-ui/react-component-ref'
```